plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/MigrateProcedure.java

```diff
@@ -300,6 +303,16 @@ private Schema toIcebergSchema(List<Column> columns)
         return new Schema(icebergSchema.asStructType().fields());
     }
 
+    private Type toIcebergType(io.trino.spi.type.Type fromType, AtomicInteger nextFieldId, HiveStorageFormat storageFormat)
+    {
+        if (fromType.equals(TimestampType.TIMESTAMP_MILLIS)) {
+            if (IcebergFileFormat.ORC.name().equals(storageFormat.name()) || IcebergFileFormat.PARQUET.name().equals(storageFormat.name())) {
```

Comment:
HiveStorageFormat is enum. We can simply use `=` instead of `equals`.

Reply:
Reminder.

Reply:
@ebyhr but here we compare value of `IcebergFileFormat` with value of `HiveStorageFormat` so I guess you can't compare them usign `==` 
The question is should `IcebergFileFormat` be used here, but I guess it makes sense

Reply:
Why do you want to compare HiveStorageFormat and IcebergFileFormat? Just comparing within HiveStorageFormat should be enough.

Reply:
I just want to make a connection with the supported Iceberg file formats.
However, I don't not insist on the current.

Reply:
> I just want to make a connection with the supported Iceberg file formats.

It's simply redundant and reducing readability. I would recommend changing the current implementation.

Reply:
Pushed